### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by johnny2194**
I have just renamed the original README to HOWTOCONTRIBUTE and created our working README and MIT license.

### CODE REVIEW

File: views/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+#READ ME
```
Comment (DewarM) [inline views/README.md]: The titles should be # READ ME as to render the markdown correctly

File: views/LICENSE.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+##License
```
Comment (DewarM) [inline views/LICENSE.md]: This title should be # License to render correctly

**Review by DewarM — CHANGES_REQUESTED** [review]
The markdown needs to be changed slightly before merging.

**Comment by johnny2194** [comment]
@DewarM Check out my fix to titles

**Review by DewarM — APPROVED** [review]
LGTM Ready to merge

**Comment by johnny2194** [comment]
@DewarM Dude check out my fix, I have not closed it this time.  FYI guys, don't choose close and comment 👍

**Comment by DewarM** [comment]
@johnny2194 your doc branch can now safely be deleted

**Pull request merged**